### PR TITLE
[PA-1537-T2] Add DeleteObjectsByMultipleUsersFromNewAdmin

### PR DESCRIPTION
**Pull request by kphalgun-px**
**What this PR does / why we need it**:
This PR adds a test case named "DeleteObjectsByMultipleUsersFromNewAdmin" to automate the use case of verifying backup, restore, schedule object from non-admin user is listed for multiple admins. This test case will increase the test coverage of Px-Backup TC Automation and ensure that the orphaned object feature functions as intended.

**Which issue(s) this PR fixes** (optional)
Closes #PA-1405

**Special notes for your reviewer**:
This test case is applicable to Px-Backup versions 2.6.0 and above.

TestRail: https://portworx.testrail.net/index.php?/cases/view/87567

Please review the following Jenkins builds:

v2.5.2

S3: I will update the results once the test run is complete.

NFS: I will update the results once the test run is complete.

### CODE REVIEW

File: tests/backup/backup_orphaned_test.go [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+					}
+				}
+			})
+		}
+		Step(fmt.Sprintf("Add new user to %s group", adminGroup), func() {
+			log.InfoD(fmt.Sprintf("Adding new user to %s group", adminGroup))
+			for _, user := range createUsers(1) {
+				err := backup.AddGroupToUser(user, adminGroup)
+				log.FailOnError(err, "failed to add user %s to the group %s", user, adminGroup)
+				newAdmin = user
```
Comment (snigdha-px) [inline tests/backup/backup_orphaned_test.go:255]: Can we just print the admin user name .It will help in debugging in case of failure

**Review by snigdha-px — APPROVED** [review]
LGTM except minor comments

File: tests/backup/backup_orphaned_test.go [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
+)
+
+// 1537-T2
+// DeleteObjectsByMultipleUsersFromNewAdmin delete backups, backup schedules, restore and cluster objects created by multiple user from the new admin
+var _ = Describe("{DeleteObjectsByMultipleUsersFromNewAdmin}", func() {
+
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/87567
```
[the remark this answers is not in the document]
Comment (kphalgun-px) [reply tests/backup/backup_orphaned_test.go:22]: Is this the formatting you would like for all the orphaned test cases?

File: tests/backup/backup_orphaned_test.go [hunk 3]
```diff
@@ -0,0 +1,422 @@
...
+		userBackupMap              = make(map[string]map[string]string)
+		userScheduleNameMap        = make(map[string]string)
+		userRestoreMap             = make(map[string]map[string]string)
+		numberOfUsers              = 2
+		numberOfBackups            = 1
+		newAdmin                   string
+		adminGroup                                     = "px-admin-group"
+		infraAdminRole             backup.PxBackupRole = backup.InfrastructureOwner
+	)
+
```
[the remark this answers is not in the document]
Comment (kphalgun-px) [reply tests/backup/backup_orphaned_test.go:43]: like this?

```
	var (
		scheduledAppContexts       = make([]*scheduler.Context, 0)
		appNamespaces              = make([]string, 0)
		infraAdminUsers            = make([]string, 0)
		providers                  = getProviders()
		userCloudCredentialMap     = make(map[string]map[string]string)
		userBackupLocationMap      = make(map[string]map[string]string)
		userClusterMap             = make(map[string]map[string]string)
		userSchedulePolicyInterval = int64(15)
		userSchedulePolicyMap      = make(map[string]map[string]string)
		userBackupMap              = make(map[string]map[string]string)
		userScheduleNameMap        = make(map[string]string)
		userRestoreMap             = make(map[string]map[string]string)
		numberOfUsers              = 2
		numberOfBackups            = 1
	)

	var (
		newAdmin       string
		adminGroup                         = "px-admin-group"
		infraAdminRole backup.PxBackupRole = backup.InfrastructureOwner
	)
```

File: tests/backup/backup_orphaned_test.go (hunk 1 above)
Comment (kphalgun-px) [reply tests/backup/backup_orphaned_test.go:255]: Done.

File: tests/backup/backup_orphaned_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (kphalgun-px) [reply tests/backup/backup_orphaned_test.go:22]: Will stick to the current formatting.

File: tests/backup/backup_orphaned_test.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (kphalgun-px) [reply tests/backup/backup_orphaned_test.go:43]: Will stick to the current formatting.

**Review by ak-px — APPROVED** [review]
LGTM

**Comment by kphalgun-px** [comment]
Thank you for the reviews. These changes are now included in PR #1663.

**Pull request closed without merging**